core/src/test/java/com/graphhopper/routing/util/EncodingManagerTest.java

```diff
@@ -66,6 +71,13 @@ public void testEncoderAcceptNoException()
     }
 
     @Test
+    public void testEncoderWithWrongVersionIsRejected()
+    {
+        thrown.expect(IllegalArgumentException.class);
```

Comment:
I can go through some existing tests and adjust them, would be a good way for me to learn more about JUnit…